docs/src/modules/java/pages/ai-coding-assistant.adoc

```diff
@@ -107,27 +107,14 @@ Make sure that you download the latest documentation regularly to make use of do
 
 === Claude code
 
-For https://www.anthropic.com/claude-code[Claude code] you need to define a `CLAUDE.md` file with the following instructions:
+For https://www.anthropic.com/claude-code[Claude code] you need to define a `CLAUDE.md` file with instructions.
 
-[source, md]
-----
-# CLAUDE.md
-
-This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
-
-## Build and Development Commands
-
-See @README.md for a project overview, and how to build, test and run the application.
+* Download link:../_attachments/CLAUDE.md[CLAUDE.md] to `CLAUDE.md` file in the project directory.
+* Download link:../_attachments/AGENTS.md[AGENTS.md] to `AGENTS.md` file in the project directory.
```

Comment:
We will incorporate this in `akka code init`, and we should rewrite this whole page to make use of `akka code init`